kafka/pdl-leesah-ingestor/src/main/kotlin/no/nav/amt/tiltak/kafka/pdl_leesah/LeesahIngestorImpl.kt

```diff
@@ -43,11 +46,23 @@ class LeesahIngestorImpl(
 				secureLog.info("Har slettet deltakere personidenter: ${it.personIdent}")
 
 			}
+		} else {
```

Comment:
Her henger jeg ikke helt med.. Skal vi bare oppdatere navn hvis du ikke er addressebeskyttet? Her må det sjekkes om navn!=null, isåfall henter man det ut og oppdaterer

Reply:
Ja, ser nå at bruker ikke slettes selvom deltakeren slettes (selvsagt in hindsight) så dette skal gjøres uansett :) Will fix!